<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="btnProcess">
       <property name="text">
        <string>Обработать</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Выгрузить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnLoad">
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>161</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Мин </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="light_min"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Светимость </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>511</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Сортировка</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="sortComboBox">
       <item>
        <property name="text">
         <string>По интенсивности
</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>По количеству пикселей
</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>По координате X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>По координате Y
</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="sortButton">
       <property name="text">
        <string>Сортировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
